src/Sidebar.js

```diff
@@ -1,10 +1,47 @@
-import React from 'react'
+import React, { useContext } from 'react'
+import { Link } from 'react-router-dom'
+
 import logo from './logo.svg'
 import { FaTimes } from 'react-icons/fa'
 import { social, links } from './data'
+import { AppContext } from './context'
 
 const Sidebar = () => {
-  return <h2>sidebar</h2>
+  const sidebarLinks = links.map((link) => {
+    return (
+      <li key={link.id}>
+        <Link to={link.url}>{link.text}</Link>
+      </li>
+    )
+  })
+
+  const { isSideBarOpen, closeSideBar } = useContext(AppContext);
```

Comment:
try to declare variables, constants at the top of the component for better readability